### PR TITLE
Add dropdown menu, closes #13

### LINKED ISSUE #13: Add updates page with "What's New"

### PR DESCRIPTION
**Pull request by orischwartz**
**Changes**
* Add dropdown menu for signed in user (for now always sign in as `"test-user", "test-ro"`)
* Add what's new page, MD5 hash contents to determine when changed and display the what's new indicator.

**Notes**
* The javascript style used here (revealing module w/ global import) isn't the same as as used in #24. Do we have a standard?
* 508 compliance testing required some changes to CSS that you wouldn't ordinarily have. Was forced to define a background color of white for the dropdown trigger (username) when we really want it to be transparent and match the header in general.

### CODE REVIEW

**Comment by orischwartz** [comment]
@shanear This is mostly a request for feedback:

* What other RSpec tests would you add?
* What javascript testing would you add? 
* Any thoughts on Javascript style here vs your #24 ?

**Comment by orischwartz** [comment]
Code coverage failing for `certifications_controller.rb` and `form8.rb`, neither of which were changed here....

**Comment by shanear** [comment]
@orischwartz yeah, i'm not sure what's happening with this code coverage business. Master is failing for code coverage as well.

I'll take a look into it.

**Comment by orischwartz** [comment]
Looks like master was failing [travis build](https://travis-ci.org/department-of-veterans-affairs/caseflow-certification/builds/109681827)

File: app/assets/javascripts/utils.js [hunk 1]
```diff
@@ -0,0 +1,24 @@
+var CFUtils = (function ($) {
```
Comment (shanear) [inline app/assets/javascripts/utils.js]: we shouldn't really be using javascript animations. css3 animations are what we want, they're way more performant, and don't require unnecessary logic in the app.

 css animations will remove the need for this file. I think this is fine for now though. I just wouldn't add any tests.

**Comment by orischwartz** [comment]
@shanear #24 passes so I think after I review that and merge here it could address it.

File: app/assets/javascripts/dropdrown.js [hunk 2]
```diff
@@ -0,0 +1,22 @@
+var CFDropdown = (function ($) {
+    // private
```
Comment (shanear) [inline app/assets/javascripts/dropdrown.js]: There are some differences between your JS style and the style in my PR (https://github.com/department-of-veterans-affairs/caseflow-certification/pull/24/files).

- the `CF` prefix on class names
Don't care here, we should just keep it consistent.

- assigning the class to the window inside of the container function, as opposed to having the container function returning the class and setting it as a var in the global namespace.
don't know what the best option here is, but again, should keep it consistent

- a) `var functionName = function() { ... }` vs b) `function functionName() { ... }`
the only reason why i like b a little more is that it gives the function a name for debugging purposes. Exceptions will tell you they occured in `functionName()` as opposed to an anonymous function.

Comment (shanear) [inline app/assets/javascripts/dropdrown.js]: Also, for testing this file, you can create a fake dropdown dom element in the test, run this function, and use jQuery to poke it and verify the result (you can look here for an example of how I do this: https://github.com/department-of-veterans-affairs/caseflow-certification/pull/24/files#diff-e70f6b825e2315cdeedf28a79580956bR32)

File: spec/feature/whats_new_spec.rb [hunk 3]
```diff
@@ -0,0 +1,16 @@
+require "rails_helper"
+
+RSpec.feature "What's New" do
+
```
Comment (shanear) [inline spec/feature/whats_new_spec.rb]: Looks great, if anything, I kindof think the first test is redundant, but it's cool with me either way.

File: app/assets/javascripts/application.js [hunk 4]
```diff
@@ -13,3 +13,10 @@
 //= require jquery
 //= require jquery_ujs
 //= require_tree .
+
+// main
```
Comment (shanear) [inline app/assets/javascripts/application.js]: this should probably happen in a script tag on the page or layouts where it's relevant.

File: app/assets/javascripts/dropdrown.js (hunk 2 above)
Comment (orischwartz) [inline app/assets/javascripts/dropdrown.js]: * Agree on function naming convention, will update.
* My thinking with return values is that it makes explicit the global dependencies and what it produces, e.g.,: 

```javascript
window.DateField = (function($){
...
})($);
```

Defines a module that depends on the global jQuery object and produces the DateField object.

* Will drop the CF prefix but if we're putting things into the global namespace or in window we don't want to clash with something else. 
* Agree on all other notes, will make necessary adjustments.

**Pull request merged**